test/univariate/continuous/noncentralbeta.jl

```diff
@@ -0,0 +1,15 @@
+using Distributions
+using Test
+
+
+@testset "NoncentralBeta" begin 
+    α = 2.0; β = 3.0; λ = 1.0
```

Comment:
Could we actually test more random/less nice values?  Or directly use the R reference setup?